plugins/modules/keycloak_component.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Björn Bösel <bjoernboesel@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_component
+
+short_description: Allows administration of Keycloak components via Keycloak API
+
+version_added: 9.5.0
```

Comment:
9.5.0 has been released, this must change to:
```suggestion
version_added: 10.0.0
```